.storybook/stories/modal/modal.stories.ts

```diff
@@ -6,12 +6,46 @@
 
 import { ClrModal, ClrModalModule, commonStringsDefault } from '@clr/angular';
 import { action } from '@storybook/addon-actions';
-import { Parameters } from '@storybook/addons';
-import { Story } from '@storybook/angular';
+import { moduleMetadata, Story, StoryObj } from '@storybook/angular';
 
-import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+import { CommonModules } from '../../helpers/common';
 
-const defaultStory: Story = args => ({
+export default {
+  title: 'Modal/Modal',
+  decorators: [
+    moduleMetadata({
+      imports: [...CommonModules, ClrModalModule],
+    }),
+  ],
+  component: ClrModal,
+  argTypes: {
+    // inputs
+    clrModalCloseButtonAriaLabel: { type: 'string', defaultValue: commonStringsDefault.close },
+    clrModalLabelledById: { defaultValue: '' },
+    clrModalSize: { defaultValue: 'md', control: { type: 'radio', options: ['sm', 'md', 'lg', 'xl'] } },
+    clrModalSkipAnimation: { defaultValue: false, control: { type: 'boolean' } },
+    // outputs
+    clrModalAlternateClose: { control: { disable: true } },
```

Comment:
It looks like our outputs do not work (and this is so on the currently deployed version of storybook)